lib/albacore/task_types/find_msbuild_versions.rb

```diff
@@ -20,6 +20,11 @@ def self.find_msbuild_versions
     rescue
       error "failed to open HKLM\\SOFTWARE\\Microsoft\\MSBuild\\ToolsVersions"
     end
+    
+    # MSBuild 15, assume default installation path
+    vs2017_dir = Dir[File.join(ENV['ProgramFiles(x86)'].gsub('\\', '/'), 'Microsoft Visual Studio', '2017', '*')].first
+    retval[15] = File.join(vs2017_dir, 'MSBuild', '15.0', 'Bin') unless vs2017_dir.nil?
+    
```

Comment:
This should not be required for a properly-installed Visual Studio 2017. The logic above this should discover MSBuild in the Visual Studio 2017 install folder. I wrote the logic above, and prior to this logic, Albacore would launch Visual Studio 2017's MSBuild and not 2019's. Did you find that the logic wasn't able to detect MSBuild 15 from Visual Studio 2017 on your system??

Reply:
Never mind, just read a line in the PR description I had glossed over in my first read through. :-) I will in fact delete these comments, but am adding this one first in case you got an e-mail about the preceding one.